Mathlib/AlgebraicGeometry/EllipticCurve/Affine/Point.lean

```diff
@@ -549,7 +549,7 @@ lemma add_self_of_Y_eq {x₁ y₁ : F} {h₁ : W.Nonsingular x₁ y₁} (hy : y
     some h₁ + some h₁ = 0 :=
   add_of_Y_eq rfl hy
 
-@[simp]
+-- @[simp] -- Not a good simp lemma, since `hy` is not in simp normal form.
```

Comment:
In that it uses Ne instead of Not? Or is there something more notable?

In general a hypothesis may be simp-normal in the general case, but not so as soon as it is specialized, so leaving this tagged simp seems at worst a mild performance cost

Reply:
No, the issue is that `simp` will unfold `negY`.
If we do not want that, I think that should be considered in a separate PR. For now, I think we should just fix the simp set.